core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2183,6 +2183,13 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Return SparkContext
+   */
+  private[spark] def get(): SparkContext = {
```

Comment:
Actually, there is already a such explicit signal: `RpcEnv.shutdown`. Previously, I was worried about the immature NettyRpcEnv. So the stack trace of the error is still logged to help debug after RpcEnv is shutdown. However, since `NettyRpcEnv` becomes mature, we can remove the stack trace.